gradle.properties

```diff
@@ -19,7 +19,7 @@ checkstyleToolVersion=8.2
 #in integration tests- same version(4.5.0) of client and server
 bookKeeperVersion=4.5.0
 commonsioVersion=2.5
-commonsLang3Version=3.6
+commonsLang3Version=3.7
```

Comment:
Is there a specific reason for upgrading here or is it convenience?

Reply:
Yes. Some features we were using for tests in Commons Lang were deprecated in 3.6 and in-deprecated in 3.7.

There's a comment I made earlier in this PR about this.